api/tests/identity-access-management/integration/domain/usecases/get-current-user.usecase.test.js

```diff
@@ -0,0 +1,105 @@
+import { usecases } from '../../../../../src/identity-access-management/domain/usecases/index.js';
+import { anonymousUserTokenRepository } from '../../../../../src/identity-access-management/infrastructure/repositories/anonymous-user-token.repository.js';
+import { CampaignParticipationStatuses } from '../../../../../src/prescription/shared/domain/constants.js';
+import { Assessment } from '../../../../../src/shared/domain/models/Assessment.js';
+import { databaseBuilder, expect } from '../../../../test-helper.js';
+
+describe('Integration | Identity Access Management | Domain | UseCase | get-current-user', function () {
+  it('gets the user basic informations', async function () {
```

Comment:
En anglais on ne met pas de _« s »_ dans ce cas : 
```suggestion
  it('gets the user basic information', async function () {
```